### PR TITLE
feat: Added Product Listing Page

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Product listing Page with filter Section.
- Mobile Responsive Navigation will be taken care in the upcoming changes.

### CODE REVIEW

**Review by Ronak-Mutha — COMMENTED** [review]
Clean code and good use of variables. Good to merge.

File: Products/product.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import url("/style.css");
+.filterList {
+    padding: 3rem 0 0 3rem;
+    flex-basis: 250px;
+    min-width: 250px;
+    height: 100vh;
```
Comment (Bhakti199) [inline Products/product.css:6]: use rem instead of px

**Review by Bhakti199 — APPROVED** [review]

File: Products/product.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Products/product.css:6]: Actually.. The point of flex-basis is to have a fixed width( not to rely on root font-size). So, I thought pixels would be the best. Please suggest an alternate way, if there is any?

**Review by nerdyspook — APPROVED** [review]
Code seems good to merge

**Pull request merged**